.github/workflows/storybook.yaml

```diff
@@ -1,14 +1,11 @@
 name: storybook
-env: 
+env:
     STORYBOOK_TOKEN: ${{ secrets.STORYBOOK_TOKEN }}
 on:
   push:
     branches:
       - master
-  pull_request:
-    branches:
-      - master
```

Comment:
I think we can leave the PR for master - the publish step should only run if its a push rather than a pull.